apps/gauzy/src/app/pages/organizations/edit-organization/edit-organization-settings/edit-organization-other-settings/edit-organization-other-settings.component.html

```diff
@@ -1508,7 +1508,7 @@ <h4 class="settings-section-header">
 					<nb-accordion-item-header>
 						{{ 'ORGANIZATIONS_PAGE.EDIT.INTEGRATIONS' | translate }}
 					</nb-accordion-item-header>
-					<nb-accordion-item-body>
+					<nb-accordion-item-body #integrationsBody [hidden]="integrationsBody?.state === 'collapsed' ? true : false">
```

Comment:
```suggestion
					<nb-accordion-item-body #integrationsBody [hidden]="integrationsBody?.state === 'collapsed'">
```
@coderNadir please simplify condition.

Reply:
@adkif noted Adolphe